_posts/2022-05-27-jay-lofstead.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "SC22 Mini-series: Inclusivity Committee"
+author: "@mmundt"
+rse: "Jay Lofstead"
+excerpt: "Jay Lofstead of Sandia National Laboratories has some big plans for the Inclusivity Committee at Supercomputing 2022."
+date: "2022-05-27 8:30:00"
```

Comment:
I think next Thursday is the 26th?
```suggestion
date: "2022-05-26 8:30:00"
```

Reply:
You are right - it is. Sorry, I think you said the 27th on Slack but also Thursday and my brain prefers numbers.